_stbt/ocr.py

```diff
@@ -240,10 +241,16 @@ def ocr(frame=None, region=Region.ALL,
         that tesseract won't think that a zero is the letter o.
         Note that Tesseract 4.0's LSTM engine ignores ``char_whitelist``.
 
+    :param dict corrections:
```

Comment:
A list of tuples may provide more flexibility.  That way the keys needn't be hashable.  I suppose we could always do:

```
if isinstance(corrections, dict):
    corrections = list(corrections.items())
```

Reply:
Interestingly it turns out that `re.compile` objects are hashable:

```python
>>> hash(re.compile("abc"))
8787775815358
```

Reply:
I prefer dicts because there's far less syntax characters involved. With Python 3 they're ordered already, and in Python 2 you can use an OrderedDict if you care about the order.

Reply:
If Python 2 users are going to need to use `OrderedDict` I think this should be explicitly called out in the documentation.  It can be tricky to get right.  For example:

Looks right at first glance, but is not:

```py
>>> collections.OrderedDict({"a": 1, "b": 2, "c": 3})
OrderedDict([('a', 1), ('c', 3), ('b', 2)])
```

Correct:

```py
>>> collections.OrderedDict([("a", 1), ("b", 2), ("c", 3)])
OrderedDict([('a', 1), ('b', 2), ('c', 3)])
```